<html>
<head><title>Keggs</title><link rel="stylesheet" href="mystyles.css"><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
<body><div id="header">
<H1>Kegg Results</H1></div><body><div>
<table border="1" style="width:100%"><tr><th>ID</th><th>Title</th><th>Name</th><th>Product</th><th>Comment</th><th>Orthology</th></tr><tr><td> EC 3.6.3.14 Enzyme</td><td> Direct observation of the rotation of F1-ATPase.</td><td> ATP phosphohydrolase (H+-transporting)</td><td> ADP [CPD:C00008];</td><td> A multisubunit non-phosphorylated ATPase that is involved in the transport of ions. Large enzymes of mitochondria, chloroplasts and bacteria with a membrane sector (Fo, Vo, Ao) and a cytoplasmic-compartment sector (F1, V1, A1). The F-type enzymes of the inner mitochondrial and thylakoid membranes act as ATP synthases. All of the enzymes included here operate in a rotational mode, where the extramembrane sector (containing 3 alpha- and 3 beta-subunits) is connected via the delta-subunit to the membrane sector by several smaller subunits. Within this complex, the gamma- and epsilon-subunits, as well as the 9-12 c subunits rotate by consecutive 120_degree_ angles and perform parts of ATP synthesis. This movement is driven by the H+ electrochemical potential gradient. The V-type (in vacuoles and clathrin-coated vesicles) and A-type (archebacterial) enzymes have a similar structure but, under physiological conditions, they pump H+ rather than synthesize ATP.</td><td> K02111 F-type H+-transporting ATPase subunit alpha</td></tr>
<tr><td> EC 6.3.4.19 Enzyme</td><td> Structural basis for translational fidelity ensured by transfer RNA lysidine synthetase.</td><td> L-lysine:[tRNAIle2]-cytidine34 ligase (AMP-forming)</td><td> [tRNAIle2]-lysidine34;</td><td> The bacterial enzyme modifies the wobble base of the CAU anticodon of tRNAIle at the oxo group in position 2 of cytidine34. This modification determines both codon and amino acid specificities of tRNAIle.</td><td> K04075 tRNA(Ile)-lysidine synthase</td></tr>
<tr><td> EC 5.1.1.3 Enzyme</td><td> Glutamic acid racemase from Lactobacillus arabinosus.</td><td> glutamate racemase</td><td> D-glutamate [CPD:C00217]</td><td> A pyridoxal-phosphate protein.</td><td> K01776 glutamate racemase</td></tr>
<tr><td> EC 2.7.1.33 Enzyme</td><td> The phosphorylation of pantothenic acid by Lactobacillus arabinosus 17-5.</td><td> ATP:(R)-pantothenate 4'-phosphotransferase</td><td> ADP [CPD:C00008];</td><td></td><td> K00867 type I pantothenate kinase</td></tr>
<tr><td> EC 2.4.2.9 Enzyme</td><td> Nucleotide synthesis from 5-phosphoribosylpyrophosphate.</td><td> UMP:diphosphate phospho-alpha-D-ribosyltransferase</td><td> uracil [CPD:C00106];</td><td></td><td> K00761 uracil phosphoribosyltransferase</td></tr>
<tr><td> EC 2.1.1.33 Enzyme</td><td> Two proteins that form a complex are required for 7-methylguanosine modification of yeast tRNA.</td><td> S-adenosyl-L-methionine:tRNA (guanine-N7)-methyltransferase</td><td> S-adenosyl-L-homocysteine [CPD:C00021];</td><td> The enzyme specifically methylates guanine46 at N7 in tRNA.</td><td> K03439 tRNA (guanine-N7-)-methyltransferase</td></tr>
<tr><td> EC 3.1.3.5 Enzyme</td><td> 5'-Nucleotidase of rat liver microsomes.</td><td> 5'-ribonucleotide phosphohydrolase</td><td> ribonucleoside [CPD:C00911];</td><td> Wide specificity for 5'-nucleotides.</td><td> K01081 5'-nucleotidase</td></tr>
<tr><td> EC 5.99.1.2 Enzyme</td><td> DNA topoisomerases.</td><td> DNA topoisomerase</td><td></td><td> These enzymes bring about the conversion of one topological isomer of DNA into another, e.g., the relaxation of superhelical turns in DNA, the interconversion of simple and knotted rings of single-stranded DNA, and the intertwisting of single-stranded rings of complementary sequences, cf. EC 5.99.1.3 DNA topoisomerase (ATP-hydrolysing).</td><td> K03163 DNA topoisomerase I</td></tr>
<tr><td> EC 2.7.1.148 Enzyme</td><td> Studies on the nonmevalonate pathway: conversion of 4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol to its 2-phospho derivative by 4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol kinase.</td><td> ATP:4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol 2-phosphotransferase</td><td> ADP [CPD:C00008];</td><td> The enzyme from Escherichia coli requires Mg2+ or Mn2+. Forms part of an alternative nonmevalonate pathway for terpenoid biosynthesis (for diagram, click here).</td><td> K00919 4-diphosphocytidyl-2-C-methyl-D-erythritol kinase</td></tr>
<tr><td> EC 4.2.1.70 Enzyme</td><td> Molecular identification of pseudouridine-metabolizing enzymes.</td><td> uracil hydro-lyase (adding D-ribose 5-phosphate; pseudouridine-5'-phosphate-forming)</td><td> pseudouridine 5'-phosphate [CPD:C01168];</td><td> The reaction it readily reversible. While the enzymes from Tetrahymena pyriformis and Agrobacterium tumefaciens produce pseudouridine 5'-phosphate the enzyme from Escherichia coli functions as a pseudouridine-5'-phosphate glycosidase in vivo [5].</td><td> K01718 pseudouridylate synthase</td></tr>
<tr><td> EC 6.1.1.5 Enzyme</td><td> The enzymic synthesis of amino acyl derivatives of ribonucleic acid. II. The preparation of leucyl-, valyl-, isoleucyl- and methionyl ribonucleic acid synthetases from Escherichia coli.</td><td> L-isoleucine:tRNAIle ligase (AMP-forming)</td><td> AMP [CPD:C00020];</td><td></td><td> K01870 isoleucyl-tRNA synthetase</td></tr>
<tr><td> EC 6.3.2.10 Enzyme</td><td> Recent advances in the formation of the bacterial peptidoglycan monomer unit.</td><td> UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine:D-alanyl-D-alanine ligase (ADP-forming)</td><td> ADP [CPD:C00008];</td><td> Involved with EC 6.3.2.4 (D-alanine---D-alanine ligase), EC 6.3.2.7 (UDP-N-acetylmuramoyl-L-alanyl-D-glutamate---L-lysine ligase) or EC 6.3.2.13 (UDP-N-acetylmuramoyl-L-alanyl-D-glutamate---2,6-diaminopimelate ligase), EC 6.3.2.8 (UDP-N-acetylmuramate---L-alanine ligase) and EC 6.3.2.9 (UDP-N-acetylmuramoyl-L-alanine---D-glutamate ligase) in the synthesis of a cell-wall peptide (click here) for diagram. This enzyme also catalyses the reaction when the C-terminal residue of the tripeptide is meso-2,4-diaminoheptanedioate (acylated at its L-centre), linking the D-Ala-D-Ala to the carboxy group of the L-centre. This activity was previously attributed to EC 6.3.2.15, which has since been deleted.</td><td> K01929 UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase</td></tr>
<tr><td> EC 2.1.1.199 Enzyme</td><td> Fine-tuning of the ribosomal decoding center by conserved methyl-modifications in the Escherichia coli 16S rRNA.</td><td> S-adenosyl-L-methionine:16S rRNA (cytosine1402-N4)-methyltransferase</td><td> S-adenosyl-L-homocysteine [CPD:C00021];</td><td> RsmH catalyses the N4-methylation of cytosine1402 and RsmI (EC 2.1.1.198) catalyses the 2'-O-methylation of cytosine1402 in 16S rRNA. Both methylations are necessary for efficient translation initiation at the UUG and GUG codons.</td><td> K03438 16S rRNA (cytosine1402-N4)-methyltransferase</td></tr>
<tr><td> EC 2.7.1.23 Enzyme</td><td> Kinases for the synthesis of coenzyme A and triphosphopyridine nucleotide.</td><td> ATP:NAD+ 2'-phosphotransferase</td><td> ADP [CPD:C00008];</td><td></td><td> K00858 NAD+ kinase</td></tr>
<tr><td> EC 3.6.1.19 Enzyme</td><td> Purification and properties of a nucleoside triphosphate pyrophosphohydrolase from red cells of the rabbit.</td><td> nucleoside-triphosphate diphosphohydrolase</td><td> nucleotide [CPD:C00215];</td><td> May be identical with EC 3.6.1.9 nucleotide diphosphatase.</td><td> K01519 inosine triphosphate pyrophosphatase</td></tr>
</tr>
</table>
</div></body></html>